### ISSUE TITLE
Filtering items: Class "App\Http\Middleware\User" not found

### ISSUE THREAD

**Issue by discern** [lavary/laravel-menu#297]
I just wanted to point out to anyone having issues with the [Filtering the items](https://github.com/lavary/laravel-menu#filtering-the-items) section for checking is a user can see the menu item based on their permissions, that you may need to use `if (\Auth::user()->can(...` instead of `if (User::get()->can(...` for this to work. As the documentation is currently written, you may receive an error (I did).

**Comment by dustingraham** [comment]
I have not looked at this myself, but if this is the case, a PR for the documentation could be useful.

**Comment by discern** [comment]
Good point. I was hoping someone could verify. It worked for me on L9, but YMMV in other versions.
